frontend/package.json

```diff
@@ -31,6 +31,7 @@
     "vite-plugin-svgr": "3.2.0"
```

Comment:
could you uninstall this plugin from dependencies?

Reply:
sure